vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -379,10 +421,13 @@ export default {
       currentWebsiteUrl: "",
       currentWebsiteIndex: -1,
       contactType: -1,
```

Comment:
Can't we take default contactType = 0?

Reply:
That's business requirement, we want to show it unselected by default. `-1` = unselected.